components/licence-header.jsx

```diff
@@ -1,30 +1,31 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import {LicenceSignIn} from './licence-sign-in';
```

Comment:
looking at the other files in the repos, we use this convention:
```suggestion
import { LicenceSignIn } from './licence-sign-in';
```